.github/workflows/deploy-dev.yml

```diff
@@ -115,6 +125,29 @@ jobs:
       WFNEWS_BASEURL: https://wfnews-api.dev.bcwildfireservices.com/
       WFNEWS_QUEUESIZE: 1500
 
+      #WFONE-PUSH-NOTIFICATION-API VARIABLES
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_URL: "jdbc:postgresql://wfone-public-mobile-db-nonprod.caw9xhtwiyin.ca-central-1.rds.amazonaws.com/wfone_int_pm"
+      WFONE_PUSH_NOTIFICATION_DATASOURCE_USERNAME: "proxy_wf1_notification_rest_int"
+      WFONE_PM_SQS_S3_BUCKET_NAME: "wfone-public-mobile-monitor-queue-bucket-int"
```

Comment:
Bucket names must be globally unique - we need to create a new bucket, rather than using the one with the same name as WFONE did